`py.typed` missing from v34.0 wheel

When attempting to use the new version in Twine, I got this error:

```
twine/commands/check.py:22: error: Skipping analyzing "readme_renderer.rst": module is installed, but missing library stubs or py.typed marker
```

It's in the sdist, but not the wheel:

```
% tar tzvf readme_renderer-34.0.tar.gz 
...
-rw-r--r--  0 di     di       3668 Mar 11 15:06 readme_renderer-34.0/readme_renderer/markdown.py
-rw-r--r--  0 di     di          0 Mar 11 15:06 readme_renderer-34.0/readme_renderer/py.typed
-rw-r--r--  0 di     di       4299 Mar 11 15:06 readme_renderer-34.0/readme_renderer/rst.py
...

% unzip -l readme_renderer-34.0-py3-none-any.whl
     1184  03-11-2022 20:11   readme_renderer/__about__.py
      638  03-11-2022 20:06   readme_renderer/__init__.py
      680  03-11-2022 20:06   readme_renderer/__main__.py
     4088  03-11-2022 20:06   readme_renderer/clean.py
     3668  03-11-2022 20:06   readme_renderer/markdown.py
     4299  03-11-2022 20:06   readme_renderer/rst.py
     1050  03-11-2022 20:06   readme_renderer/txt.py
      638  03-11-2022 20:06   readme_renderer/integration/__init__.py
     3914  03-11-2022 20:06   readme_renderer/integration/distutils.py
...
```

--- a/setup.py
+++ b/setup.py
@@ -57,6 +57,7 @@ setuptools.setup(
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
     install_requires=["bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"],
+    include_package_data=True,
     entry_points={
         "distutils.commands": ["check = readme_renderer.integration.distutils:Check"],
     },

